Sources/BraintreeCore/BTAPIClient.swift

```diff
@@ -93,19 +93,16 @@ import Foundation
     /// cached on subsequent calls for better performance.
     @_documentation(visibility: private)
     public func fetchOrReturnRemoteConfiguration(_ completion: @escaping (BTConfiguration?, Error?) -> Void) {
-        configurationLoader.getConfig { [weak self] configuration, error in
-            guard let self else {
-                completion(nil, BTAPIClientError.deallocated)
-                return
-            }
-            
-            if let error {
+        // TODO: - Consider updating all feature clients to use async version of this method?
```

Comment:
Created DTMOBILES-980 for this TODO. Want to focus on tackling the crash first, then we can follow up with this change